### PR TITLE
fix #1093, powerkvm, diskfull, enable the pre-script log can be saved to the CN

### LINKED ISSUE #1093: [DEV] diskfull installation, the pre-install log can't be saved in CN for PowerKVM
After the provision, in the CN node.
The output looks like:
```
sleep 35
/opt/xcat/xcatinfo generated
trying to download postscripts from http://10.3.17.14/install/postscripts/
postscripts downloaded successfully
trying to download precreated mypostscript file http://10.3.17.14/tftpboot/mypostscripts/mypo
stscript.c910f03c17k15
failed to download precreated mypostscript, trying to generate with getpostscript.awk
generate mypostscript file successfully
/xcatpost/mypostscript.post generated
/etc/init.d/xcatpostinit1 generated
service xcatpostinit1 enabled
/opt/xcat/xcatinstallpost generated
/opt/xcat/xcatdsklspost generated
running mypostscript
Thu May 12 01:11:43 EDT 2016 Running postscript: syslog
Thu May 12 01:11:43 EDT 2016 postscript syslog return with 0
Thu May 12 01:11:43 EDT 2016 Running postscript: remoteshell
Thu May 12 01:11:47 EDT 2016 postscript remoteshell return with 0
Thu May 12 01:11:47 EDT 2016 Running postscript: syncfiles
Thu May 12 01:11:47 EDT 2016 postscript syncfiles return with 0
mypostscript returned
/boot/grub/grub.conf updated
finished node installation, reporting status...
running /xcatpost/mypostscript.post
Thu May 12 01:13:53 EDT 2016 Running postscript: otherpkgs
Thu May 12 01:13:53 EDT 2016 postscript otherpkgs return with 0
node booted, reporting status...
/xcatpost/mypostscript.post return
service xcatpostinit1 disabled
```
It lacks of some getinstdisk script output from pre-install script.

### LINKED ISSUE #1300: [DEV] diskfull installation, xcatdebugmode=2, can't access the installer via “ssh” for PowerKVM
The environment:
MN: c910f03c17k14
CN: c910f03c17k16
How to reprocedure:
```
root@c910f03c17k14:~# nodeset c910f03c17k16 osimage=pkvm3.1.0-ppc64le-install-compute
c910f03c17k16: install pkvm3.1.0-ppc64le-compute
root@c910f03c17k14:~# rpower c910f03c17k16 boot
c910f03c17k16: on reset
```
Try to access the installer via ''ssh"
```
root@c910f03c17k14:~# ssh c910f03c17k16
Warning: Permanently added 'c910f03c17k16,10.3.17.16' (ECDSA) to the list of known hosts.
root@c910f03c17k16's password:
Permission denied, please try again.
```

### PR DESCRIPTION
**Pull request by penguhyang**
fix #1093
For PowerKVM diskfull installation, enable the pre-script log can be saved to the CN
fix #1300 
For PowerKVM diskfull installation, enable access the installer via ssh when xcatdebugmode=2

### CODE REVIEW

**Comment by immarvin** [comment]
should ssh access can be enabled if site.xcatdebugmode=2?

**Comment by penguhyang** [comment]
@immarvin Thanks for review.
Yes, access installer via ssh should enabled when xcatdebugmode=2 for powerkvm.

File: xCAT-server/share/xcat/install/scripts/pre.pkvm3 [hunk 1]
```diff
@@ -18,4 +23,10 @@ fi
 echo "part PV.01 --ondisk=$instdisk" >> /tmp/partitioning
 echo "volgroup ibmpkvm_rootvg PV.01" >> /tmp/partitioning	
 
+if [ "$XCATDEBUGMODE" = "2" ]; then
```
Comment (immarvin) [inline xCAT-server/share/xcat/install/scripts/pre.pkvm3]: can the "chpasswd"  be moved to the beginning of the script?

Comment (penguhyang) [inline xCAT-server/share/xcat/install/scripts/pre.pkvm3]: Sure.

**Comment by immarvin** [comment]
agree to merge

**Pull request merged**